doc/user_manual/rom.tex

```diff
@@ -2922,8 +2922,6 @@ \subsubsection{ARMA}
 \end{equation*}
 where $x$ is a vector of dimension $n$, and $\phi_i$ and $\theta_j$ are both $n$ by $n$ matrices. When $q=0$, the above is
 autoregressive (AR); when $p=0$, the above is moving average (MA).
-While plans for optimizing $p$ and $q$ are under consideration, currently setting $Pmin$ and $Pmax$ to the
-same value is required, and similarly for $Qmin$ and $Qmax$.
 %The user is allowed to provide upper and lower limits for $p$
 %and $q$ (see below), and the training process will choose the optimal $p$ and $q$ that fall into the user-specified range.
```

Comment:
These two lines are not needed, right?

Reply:
It's likely to be added again in the future, but I can remove them for now.